packages/atomic/src/components/search/search-box-suggestions/atomic-search-box-instant-results/atomic-search-box-instant-results.tsx

```diff
@@ -90,6 +89,20 @@ export class AtomicSearchBoxInstantResults implements BaseResultList {
     }
   }
 
+  private getLink(el: HTMLElement) {
+    return el?.querySelector('atomic-result')?.shadowRoot?.querySelector('a');
```

Comment:
possible that there are more than 1 links

Reply:
yeah, I thought about that but didn't know how to go about this... maybe look for the atomic-link instead? and then any other links that don't stop propagation should already be good!